extensions/arc/src/models/controllerModel.ts

```diff
@@ -157,6 +158,7 @@ export class ControllerModel {
 	 * @param name The name of the resource
 	 */
 	public async miaaDelete(namespace: string, name: string): Promise<void> {
+		await this._registrationRouter.apiV1RegistrationNsNameIsDeletedDelete(namespace, name, true);
```

Comment:
Don't think this line belongs.  Already handled by deleteRegistration.

Reply:
Ah shoot, yeah this was left over from my attempts last week...